src/Controller/MenuController.java

```diff
@@ -162,6 +167,23 @@
 	private GridPane mainContent;
 	@FXML
 	private HBox topBox;
+	//chat variables
+	private final BorderPane mainPane = new BorderPane();
+	private final GridPane firstPane = new GridPane();
+	private final GridPane userMessagePane = new GridPane();
+	private final HBox spacingBox = new HBox();
+	private TextField tfName = new TextField("");
+	private TextField tfHost = new TextField("");
+	private TextField tfMessageToSend = new TextField();
+	private TextArea msgArea = new TextArea();
+	private final Label name = new Label("Name:");
+	private final Label host = new Label("Host:");
+	private final Button submitButton = new Button("Submit");
+	private final Button sendButton = new Button("Send");
+
+	private String suserName;
```

Comment:
I suspect that these variables should be called `sUserName` and `sHostName` (notice the second letter is capitalized).